### DISCUSSION TITLE
Black Line

### DISCUSSION THREAD

**Discussion by davonnep** [pawelsalawa/sqlitestudio#5113] [General]
### Details
Here you describe your problem details, or proposal for enhancement.
Downloaded SQL Lite Studio, provided database, and I am seeing this black line (unable to see the code i write until i go to the next line):

<img width="981" alt="Screenshot 2024-11-22 at 6 08 07 AM" src="https://github.com/user-attachments/assets/33ef4a70-324e-4977-85f9-648118b57163">

### Steps to reproduce: (no clue on this)
This applies only to bug reports. If this is a feature request, delete this section.

### Operating system: Apple MacOS
This applies only to bug reports. If this is a feature request, delete this section.

### SQLiteStudio version: v3.4.5
This applies only to bug reports. If this is a feature request, delete this section.

**Comment by pawelsalawa** [comment]
Go to Configuration, there to Look & Feel and there to Code colors. Click 'Reset to default'.

**Reply by davonnep** [reply]
Hey it still would give me the problem (once i hit 'Reset to default' it would flicker off the black line and put it back on.)

<img width="872" alt="Screenshot 2024-11-22 at 7 07 05 AM" src="https://github.com/user-attachments/assets/3a429952-f6a5-41f7-990a-f0b0054ceba1">

**Reply by pawelsalawa** [reply]
How about if you select the "Current query background" and set its color to some reasonable value - are you able to make it go away?

**Reply by davonnep** [reply]
Still the same result, it worked in the preview (i did hit Apply then Ok)

<img width="1090" alt="Screenshot 2024-11-22 at 11 17 25 AM" src="https://github.com/user-attachments/assets/ce01f084-81c3-4afa-85e9-9da511f1012d">

**Reply by davonnep** [reply]
I think I just fixed it by doing the current line background and current query background. I'll definitely try coding on it later today, and give updates

<img width="1182" alt="Screenshot 2024-11-22 at 11 21 37 AM" src="https://github.com/user-attachments/assets/a69d99d2-cd2a-4a77-af8a-4ced65d152f6">

**Reply by pvonmoradi** [reply]
I had the same problem on Letos v4.0.3 (AppImage build) on Debian. Had to change the background colors of both current line and current query. I wonder why they are set to black by default...

**Reply by pawelsalawa** [reply]
Which Style are you using? Were any of your color setting customized or all unchecked (left as default)?

**Reply by pvonmoradi** [reply]
Style is `Fusion`, and all of them were unchecked (left as default). If I click `Reset to defaults` button, it changes it to black again. Actually most of the colors would be set to black if I hit `Reset to defaults`.

This screenshot is the result of hitting `Reset to defaults` and checking all checkboxes so the colors would be visible in this screenshot:

<img width="1421" height="943" alt="image" src="https://github.com/user-attachments/assets/8ac8ada3-ed09-40eb-ab79-33e98e4dd673" />

**Reply by pawelsalawa** [reply]
What is your desktop UI? KDE? Gnome? Other? And what theme (dark/light) do you use in your system UI?

**Reply by pvonmoradi** [reply]
No standard DE, just i3wm. GTK theme is default Adwaita. I use `QT_QPA_PLATFORMTHEME="qt5ct"` for QT apps. In one of the apps (Qt6 Configuration tool) that make QT6 apps follow gtk general style, Style is set to `kvantum`.

**Reply by pvonmoradi** [reply]
By the way, I understand my setup is a bit unconventional. I just manually set colors and it's fine for me. Don't put too much time into this issue 😅.
Thanks for the software!

**Reply by pawelsalawa** [reply]
Yeah... "a bit unconventional" is little to say here :) It's hard to guarantee proper behavior in this case. I'm glad though you're fine with configured settings!
